pkg/monitor/pod.go

```diff
@@ -597,45 +569,14 @@ func lastContainerTimeFromStatus(current *corev1.ContainerStatus) time.Time {
 	return time.Time{}
 }
 
-func conditionsForTransitioningContainer(pod *corev1.Pod, current, previous *corev1.ContainerStatus, init bool, reason, cause, message string, currentTime time.Time, lastContainerTime time.Time) []monitorapi.Condition {
```

Comment:
the different cases here didn't provide value. They exist from before we started building a history of pod lifecycle in true intervals after the instants were reported.